tutorials/01_Basic_QA_Pipeline.ipynb

```diff
@@ -224,188 +154,149 @@
+    }
+   }
   },
   {
    "cell_type": "markdown",
-   "metadata": {},
    "source": [
-    "## Initialize Retriever, Reader & Pipeline\n",
-    "\n",
-    "### Retriever\n",
-    "\n",
-    "Retrievers help narrowing down the scope for the Reader to smaller units of text where a given question could be answered.\n",
-    "They use some simple but fast algorithm.\n",
-    "\n",
-    "**Here:** We use Elasticsearch's default BM25 algorithm\n",
-    "\n",
-    "**Alternatives:**\n",
-    "\n",
-    "- Customize the `BM25Retriever`with custom queries (e.g. boosting) and filters\n",
-    "- Use `TfidfRetriever` in combination with a SQL or InMemory Document store for simple prototyping and debugging\n",
-    "- Use `EmbeddingRetriever` to find candidate documents based on the similarity of embeddings (e.g. created via Sentence-BERT)\n",
-    "- Use `DensePassageRetriever` to use different embedding models for passage and query (see Tutorial 6)"
-   ]
+    "2. Convert the files you just downloaded into Haystack [Document objects](https://docs.haystack.deepset.ai/docs/documents_answers_labels#document) to write them into the DocumentStore. Apply the `clean_wiki_text` cleaning function to the text."
+   ],
+   "metadata": {
+    "collapsed": false
+   }
   },
   {
    "cell_type": "code",
    "execution_count": null,
    "metadata": {},
    "outputs": [],
    "source": [
-    "from haystack.nodes import BM25Retriever\n",
-    "\n",
-    "retriever = BM25Retriever(document_store=document_store)"
+    "from haystack.utils import clean_wiki_text, convert_files_to_docs\n",
+    "docs = convert_files_to_docs(\n",
+    "    dir_path=doc_dir,\n",
+    "    clean_func=clean_wiki_text,\n",
+    "    split_paragraphs=True\n",
+    ")"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "3. Write these Documents into the DocumentStore."
+   ],
+   "metadata": {
+    "collapsed": false
+   }
+  },
   {
    "cell_type": "code",
    "execution_count": null,
-   "metadata": {
-    "pycharm": {
-     "is_executing": false,
-     "name": "#%%\n"
-    }
-   },
    "outputs": [],
    "source": [
-    "# Alternative: An in-memory TfidfRetriever based on Pandas dataframes for building quick-prototypes with SQLite document store.\n",
-    "\n",
-    "# from haystack.nodes import TfidfRetriever\n",
-    "# retriever = TfidfRetriever(document_store=document_store)"
-   ]
+    "# Now, let's write the dicts containing documents to our DB.\n",
+    "document_store.write_documents(docs)"
+   ],
+   "metadata": {
+    "collapsed": false
+   }
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "While the default code in this tutorial uses Game of Thrones data, you can also supply your own. So long as your data adheres to the [input format](https://docs.haystack.deepset.ai/docs/document_store#input-format) or is cast into a [Document object](https://docs.haystack.deepset.ai/docs/documents_answers_labels#document), it can be written into the DocumentStore."
```

Comment:
I'd not advertise the use of 'bare' dictionaries instead of `Document` objects right away in Tut1. Users should use Document objects to write data in the docstore. The fact that it can be done with dicts as well is more of a legacy support feature than the proper way to go.

PS: this also means that I'm not fond of how https://docs.haystack.deepset.ai/docs/document_store#input-format doesn't even mention `Document` objects as a way to write documents in the Docstore :scream: (I've noticed just now).

Reply:
Ok good to know. I can shift the emphasis towards Documents instead of dicts